packages/gatsby-parallel-runner/src/processor-queue/implementations/google-functions/index.js

```diff
@@ -18,7 +18,7 @@ class GoogleFunctions {
     const config =
       googleConfig ||
       JSON.parse(fs.readFileSync(process.env.GOOGLE_APPLICATION_CREDENTIALS))
-    this.subName = `gatsby-sub-${new Date().getTime()}`
+    this.subName = `gatsby-sub-${Date().now()}`
```

Comment:
This line is giving an error, i think it should be Date.now()
(node:5800) UnhandledPromiseRejectionWarning: TypeError: Date(...).now is not a function

Reply:
`Date.now()`